packages/fuzzer/fuzzing_async.cpp

```diff
@@ -66,6 +68,14 @@ using FinalizerDataType = void;
 
 TSFN gTSFN;
 
+const std::string SEGFAULT_ERROR_MESSAGE = "Segmentation fault found in fuzz target";
+
+std::jmp_buf errorBuffer;
+
+void ErrorSignalHandler(int signum) {
+  std::longjmp(errorBuffer, signum);
```

Comment:
Could you please add a comment describing when this handler will be called, where `longjmp` jumps to and why that's required?